### PR TITLE
Add absence matcher

### PR DESCRIPTION
**Pull request by pedroadame**
Matcher for `validate_absence_of` validator.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
-### 4.0.1 (Next)
+### 4.0.2 (Next)
 
 * Your contribution here.
 
+### 4.0.1 (4/4/2018)
```
Comment (dblock) [inline CHANGELOG.md]: Can you please put this back? Rebase against master and leave versioning alone - we bump when we do a release? Thanks!

**Comment by dblock** [comment]
Does this not work with older mongoids?

**Comment by pedroadame** [comment]
Older mongoids do not provide absence validator.

**Comment by pedroadame** [comment]
Sure will do :)

File: CHANGELOG.md (hunk 1 above)
Comment (pedroadame) [reply CHANGELOG.md]: Sure will do :)

**Comment by pedroadame** [comment]
Any update on this @dblock ?

**Comment by dblock** [comment]
LGTM, needs a README update?

File: README.md [hunk 2]
```diff
@@ -290,6 +290,7 @@ end
 
 RSpec.describe Article do
   it { is_expected.to validate_length_of(:title).within(8..16) }
+  it { is_expected.to validate_absence_of(:deletion_date) }
```
Comment (pedroadame) [inline README.md]: Includes an example in README :)

Comment (dblock) [reply README.md]: OMG, Sorry

**Comment by dblock** [comment]
Merged

**Pull request merged**